static/app/components/modals/widgetViewerModal.tsx

```diff
@@ -317,11 +327,24 @@ function WidgetViewerModal(props: Props) {
                 // @ts-ignore getModel() is private but we need this to retrieve datetime values of zoomed in region
                 const model = chart.getModel();
                 const {startValue, endValue} = model._payload.batch[0];
-                const start = getUtcDateString(moment.utc(startValue));
-                const end = getUtcDateString(moment.utc(endValue));
+                const newStart = getUtcDateString(moment.utc(startValue));
+                const newEnd = getUtcDateString(moment.utc(endValue));
                 setModalSelection({
                   ...modalSelection,
-                  datetime: {...modalSelection.datetime, start, end, period: null},
+                  datetime: {
+                    ...modalSelection.datetime,
+                    start: newStart,
+                    end: newEnd,
+                    period: null,
+                  },
+                });
+                router.replace({
```

Comment:
Should we do `router.push` so that you can zoom back out of a chart after zooming in?

Reply:
👍  updated to router.push and made some adjustments to support back button zooming out